src/scripts/components/widgets/modals/ModalCreatePassword.js

```diff
@@ -0,0 +1,146 @@
+/* @flow */
+// import paratii from 'utils/ParatiiLib'
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import Title from 'components/foundations/Title'
+import TextField from 'components/widgets/forms/TextField'
+import Text from 'components/foundations/Text'
+import Button from 'components/foundations/Button'
+import NotepadLockSvg from 'components/foundations/svgs/NotepadLockSvg'
+import { ModalContentWrapper, ModalScrollContent } from './Modal'
+import {
+  PASSWORD_TEMP,
+  NEW_ACCOUNT,
+  RESTORE_ACCOUNT
+} from 'constants/ParatiiLibConstants'
+import { MODAL } from 'constants/ModalConstants'
+
+type Props = {
+  openModal: String => void,
+  closeModal: () => void,
+  secureKeystore: String => void,
+  getContext: String
+}
+
+const Footer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 50px;
+  width: 100%;
+`
+
+const ButtonContainer = styled.div`
+  margin-left: 10px;
+`
+
+const Icon = styled.div`
+  height: 180px;
+  margin: 40px 0 54px;
+  width: 100%;
+`
+
+class ModalSetPassword extends Component<Props, Object> {
+  setPassword: () => void
+  secureAccount: () => void
+  handleInputChange: (input: string, e: Object) => void
+
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      password: '',
+      confirm: '',
+      error: ''
+    }
+    this.setPassword = this.setPassword.bind(this)
+    this.secureAccount = this.secureAccount.bind(this)
+    this.handleInputChange = this.handleInputChange.bind(this)
+  }
+
+  secureAccount () {
+    this.props.openModal(MODAL.SECURE)
+  }
+
+  setPassword () {
+    if (this.state.password === this.state.confirm) {
+      if (this.props.getContext === NEW_ACCOUNT) {
+        sessionStorage.setItem(PASSWORD_TEMP, this.state.password)
+        this.props.openModal(MODAL.SHOW_SEED)
+      } else if (this.props.getContext === RESTORE_ACCOUNT) {
+        this.props.secureKeystore(this.state.password)
+        this.props.closeModal()
+      }
+    } else {
+      // Error, the two Passwords are different
+      this.setState({
+        error: `Hey, you have insert two different Passwords`
+      })
+    }
+  }
+
+  handleInputChange (input: string, e: Object) {
+    this.setState({
+      [input]: e.target.value,
+      error: ''
+    })
+  }
+
+  render () {
+    return (
+      <ModalContentWrapper>
+        <ModalScrollContent>
+          <Title>Create a password for securing your account</Title>
+          <Text small gray>
+            It will work for a password for fast transactions,{' '}
+            <strong>but only if you are logged in</strong>
```

Comment:
What does this mean? I think we'll want to clean up the verbiage a bit here. Right now it is confusing

Reply:
i've just copied the copy in the @pedrocasa UI!